Examples/Widgets/HospitalBedCapacity/src/components/HospitalWidget.ts

```diff
@@ -123,7 +157,6 @@ export default class Hospitals extends LitElement {
       (results: any, status: google.maps.GeocoderStatus) => {
         if (status === "OK") {
           this.hospitalAddress = results[0].formatted_address;
-
           const addressDetails = results[0].address_components.reduce(
             (seed: any, obj: any) => (
               obj.types.forEach((t: any) => (seed[t] = obj.short_name)), seed
```

Comment:
I'm confused about the `, seed` part at the end of this line . . . . seems like it is outside of the callback, but not a parameter of an outer function?

Reply:
I think we have an opportunity to improve the typing too, but that little bit was a wedge in my attempt